src/v2/ion/i18nTransformer.js

```diff
@@ -60,25 +60,29 @@ const I18N_OVERRIDE_MAPPINGS = {
 
   'identify-recovery.identifier': 'password.forgot.email.or.username.placeholder',
 
+  'select-authenticator-enroll.authenticator.duo': 'factor.duo',
+  'select-authenticator-enroll.authenticator.google_otp': 'oie.google_authenticator.label',
   'select-authenticator-enroll.authenticator.okta_email': 'oie.email.label',
   'select-authenticator-enroll.authenticator.okta_password': 'oie.password.label',
-  'select-authenticator-enroll.authenticator.phone_number': 'oie.phone.label',
-  'select-authenticator-enroll.authenticator.webauthn': 'oie.webauthn.label',
-  'select-authenticator-enroll.authenticator.security_question': 'oie.security.question.label',
   'select-authenticator-enroll.authenticator.okta_verify': 'oie.okta_verify.label',
-  'select-authenticator-enroll.authenticator.google_otp': 'oie.google_authenticator.label',
+  'select-authenticator-enroll.authenticator.phone_number': 'oie.phone.label',
   'select-authenticator-enroll.authenticator.rsa_token': 'factor.totpHard.rsaSecurId',
+  'select-authenticator-enroll.authenticator.security_question': 'oie.security.question.label',
+  'select-authenticator-enroll.authenticator.symantec_vip': 'factor.totpHard.symantecVip',
+  'select-authenticator-enroll.authenticator.webauthn': 'oie.webauthn.label',
 
+  'select-authenticator-authenticate.authenticator.duo': 'factor.duo',
+  'select-authenticator-authenticate.authenticator.google_otp': 'oie.google_authenticator.label',
   'select-authenticator-authenticate.authenticator.okta_email': 'oie.email.label',
   'select-authenticator-authenticate.authenticator.okta_password': 'oie.password.label',
-  'select-authenticator-authenticate.authenticator.phone_number': 'oie.phone.label',
-  'select-authenticator-authenticate.authenticator.webauthn': 'oie.webauthn.label',
-  'select-authenticator-authenticate.authenticator.security_question': 'oie.security.question.label',
-  'select-authenticator-authenticate.authenticator.okta_verify.signed_nonce': 'oie.okta_verify.signed_nonce.label',
   'select-authenticator-authenticate.authenticator.okta_verify.push': 'oie.okta_verify.push.title',
+  'select-authenticator-authenticate.authenticator.okta_verify.signed_nonce': 'oie.okta_verify.signed_nonce.label',
   'select-authenticator-authenticate.authenticator.okta_verify.totp': 'oie.okta_verify.totp.title',
-  'select-authenticator-authenticate.authenticator.google_otp': 'oie.google_authenticator.label',
+  'select-authenticator-authenticate.authenticator.phone_number': 'oie.phone.label',
   'select-authenticator-authenticate.authenticator.rsa_token': 'factor.totpHard.rsaSecurId',
+  'select-authenticator-authenticate.authenticator.security_question': 'oie.security.question.label',
+  'select-authenticator-authenticate.authenticator.symantec_vip': 'factor.totpHard.symantecVip',
+  'select-authenticator-authenticate.authenticator.webauthn': 'oie.webauthn.label',
```

Comment:
Noticed some of label have the key for oie some of them are reusing the v1 key? Do we want to keep them consistent?

Reply:
We **don't** want to be consistent here.

If there is an existing **non-OIE** key, it should be used. Reusing the existing key allows customers who have [i18n customizations](https://github.com/okta/okta-signin-widget#language-and-text) to migrate without a breaking change and/or updating their translations.